### ISSUE TITLE
Modifiers which directly model CSS properties could include a link to MDN in their KDoc

### ISSUE THREAD

**Issue by okarmazin** [varabyte/kobweb#605]
Currently there is no KDoc on these CSS-modifiers.
A link to the Mozilla docs would come in handy.

E.g. `Modifier.backgroundColor` would link to https://developer.mozilla.org/en-US/docs/Web/CSS/background-color

**Comment by bitspittle** [comment]
This was intentional as we are still giving ourselves permission to find the right shape for these kinds of CSS methods. We are probably at a good place now but not guaranteed; we've had to revisit decisions made already. Worrying about updating 1000 header docs at the same time is something we're happy to avoid until 1.0 when we've locked things down.

Users can quickly figure out how to map modifier names to css properties for now.
